python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -7,73 +7,37 @@ The importance factors derived from a Taylor expansion are
 defined to rank the sensitivity of the inputs to
 the output for central dispersion analysis.
 
-Let us denote by :math:`\uX` the input random vector.
-Assume that the marginals of :math:`\uX` are independent.
-Suppose that :math:`Z = h(\ux)` is a real function of the
-input, i.e. :math:`n_Z = 1`.
-Assume that the order 1 Taylor expansion of the function :math:`h`
-at the point :math:`\ux = \muX` is exact, i.e.:
+We denote by :math:`\uX` the input random vector. We assume that the marginals of :math:`\uX` are independent.
 
-.. math::
-    h(\ux) = h(\muX)
-    + \sum_{i=1}^{n_X} \frac{\partial h}{\partial x_i} (\muX) (x_i - \mu_i)
-
-where:
+We note :math:`Y = h(\ux)` with :math:`h: \Rset^d \rightarrow \Rset` a scalar function.
 
--  :math:`\muX` is the mean of the input random vector,
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations.
 
--  :math:`\frac{\partial h}{\partial x_i} (\muX)`
-   is the partial derivative of the model :math:`h`
-   with respect to the i-th input variable,
-   evaluated at the point :math:`\ux`.
-
-Therefore the expectation of :math:`Z` is:
+The first order  Taylor expansion of :math:`h` around :math:`\vect{\mu} = \Expect{\uX}` yields to the first order approximation of :math:`\Expect{Y}`:
```

Comment:
This equation is already written in _taylor_expansion_moments, so I suggest to remove it from the current page because it is off-topic. The current page focuses on the variance, not the expectation.
```suggestion
```

Reply:
yes